src/integrationtest/resources/features/download.feature

```diff
@@ -0,0 +1,26 @@
+@DownloadFeature
+Feature: Download test
```

Comment:
There is no test for the endpoint /download?

Reply:
In the controller only '/download/*' is defined. I don't think there is support for bridging the file list.